.circleci/config.yml

```diff
@@ -223,9 +223,16 @@ commands:
   visual_regression_tests:
     description: Run Percy Visual Regression Tests
     steps:
-      - run_command:
-          command_description: Run Percy visual regression tests for packages
-          command_name: "test:visual"
+      - run:
+          name: Run Percy visual regression tests for packages
+          command: |
+            val=$(node ./visual-regression-preflight.js)
+            if [ $val == "true" ];
+            then
+              yarn test:visual
+            else
+              echo Skipping visual regression tests
```

Comment:
Could this message include why they are being skipped?  Something like `Skipping visual regression tests as PR is either draft or WIP.`